doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_postgis_sink.md

```diff
@@ -0,0 +1,29 @@
+# <a name="top"></a>NGSIPostGISSink
+Content:
+
+* [Functionality](#section1)
+* [Administration guide](#section2)
+* [Programmers guide](#section3)
+
+## <a name="section1"></a>Functionality
+`com.iot.telefonica.cygnus.sinks.NGSIPostGISSink`, or simply `NGSIPostGISSink` is a sink designed to persist NGSI-like context data events within a [PostGIS server](https://postgis.net/) which is relational database exteion of [PostgreSQL server](https://www.postgresql.org/) which allows store GIS objects (Geographic Information Systems). Usually, such a context data is notified by a [Orion Context Broker](https://github.com/telefonicaid/fiware-orion) instance, but could be any other system speaking the <i>NGSI language</i>.
```

Comment:
Typo: exteion

Reply:
fixed in 3bd613cdfb7d088d123ffd96fac8e0975eb9d9c7